### PR TITLE
Create sample extension with Shell side navigation

### PR DESCRIPTION
**Pull request by kistll**

### CODE REVIEW

File: samples/with-shell-navigation/index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+      <div id="extension-main" class="extension-main">
+        <section id="section-home" class="section-hidden">
+          <h2 class="fd-title fd-title--h2" id="home-title"></h1>
```
Comment (biiit) [inline samples/with-shell-navigation/index.html:64]: start tag is `h2` but end tag is `h1`

File: samples/with-shell-navigation/index.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          <p id="home-text"></p>
+          <a id="link-to-edit-text" href="#/actions/edit" onclick="syncWithLuigi('/actions/edit')"></a>
+        </section>
+        <section id="section-actions-edit" class="section-hidden">
+          <h2 class="fd-title fd-title--h2" id="actions-edit-title"></h1>
```
Comment (biiit) [inline samples/with-shell-navigation/index.html:69]: same as above

File: samples/with-shell-navigation/index.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+          <p id="actions-edit-text"></p>
+          <a id="link-to-home-text" href="#/" onclick="syncWithLuigi('/home')"></a>
+        </section>
+        <section id="section-not-found" class="section-hidden">
+          <h2 class="fd-title fd-title--h2" id="not-found-title"></h1>
```
Comment (biiit) [inline samples/with-shell-navigation/index.html:74]: same as above

**Review by biiit — APPROVED** [review]
LGTM

**Comment by sebastienbarbier** [comment]
I would move the `<td><a href="samples/with-shell-navigation/">Inside Shell with Shell navigation</a></td>` from README within the front-end architecture table I guess :). Not sure this is the best way to sort extensions yet but let's try for now :). @kistll  can you make the change ? :)

**Review by sebastienbarbier — APPROVED** [review]

**Pull request merged**